client/src/store/new-request-module.ts

```diff
@@ -1098,7 +1118,7 @@ export class NewRequestModule extends VuexModule {
 
   get entityTypeOptions () {
     let bcOptions: SelectOptionsI[] = this.entityTypesBC.filter(x => {
-      if (this.request_action_cd === 'REH' && this.location === 'BC') {
+      if ((['MVE', 'REH'].includes(this.request_action_cd) && this.location === 'BC')) {
```

Comment:
Maybe another comment here.
(Explain the business logic that this code is implementing.)

Reply:
For sure!

Reply:
I'm not so sure of the business rules behind this, i think this is just a short list preference based on what is the most popular selections according to business?

Reply:
Well... what is this code achieving? Why did you change it?

Reply:
Good comment, thanks.

Reply:
haha mainly because this document said too :) 

https://docs.google.com/spreadsheets/d/1rIv-AN59jfsHsBIvqKa9ZRFQ63O5aQaG-ol5HWLYxZ0/edit#gid=1235129778